Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/ListView.cs

```diff
@@ -26,5 +26,28 @@ public static SeparatorStyle GetSeparatorStyle(this IPlatformElementConfiguratio
 			SetSeparatorStyle(config.Element, value);
 			return config;
 		}
+
+		public static readonly BindableProperty GroupHeaderStyleProperty = BindableProperty.Create(nameof(GroupHeaderStyle), typeof(GroupHeaderStyle), typeof(FormsElement), GroupHeaderStyle.Plain);
```

Comment:
We should make a note this will only work when setting once, since we are not handling if this property changes on the iOS platform. I do think it doesn't make sense to change this after it's being applied the first time.

Reply:
@rmarinho agreed.
What is your proposition? I thought about it, but have no idea, how to make it more user-friendly.
Just leave a doc-comment something like

```csharp
        /// <summary>
        /// You should change this value during setting up a list. Changing this value won't affect style after list rendering.
        /// </summary>
```